task/04-arrays-tasks.js

```diff
@@ -353,7 +404,15 @@ function getItemsSum(arr) {
  *  [ null, undefined, NaN, false, 0, '' ]  => 6
  */
 function getFalsyValuesCount(arr) {
-   throw new Error('Not implemented');
+   let count = arr.length;
+   arr.filter(function(elem){
+      if (elem){
```

Comment:
it is necessary to observe indents near bracket
https://javascript.info/coding-style